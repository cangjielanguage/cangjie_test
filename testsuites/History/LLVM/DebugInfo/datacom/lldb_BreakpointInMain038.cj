// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_BreakpointInMain038_expected.txt ../basecase/013_debug_lambda.cj
// EXEC: cjc -g 013_debug_lambda.cj
// EXEC: python3 DatacomTest.py lldb_BreakpointInMain038_expected.txt



