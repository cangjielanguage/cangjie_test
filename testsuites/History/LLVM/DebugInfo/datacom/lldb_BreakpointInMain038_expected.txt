lldb,lldb 013_debug_lambda,013_debug_lambda
b,b 4,Breakpoint
b,b 5,013_debug_lambda.cj:6
b,b 6,013_debug_lambda.cj:6
b,b 7,013_debug_lambda.cj:8
b,b 8,013_debug_lambda.cj:8
r,r,-> 4
c,c,-> 8
n,n,-> 6
n,n,-> 8
n,n,-> 9
cj_plocals,cj_plocals,lamdba001 = 20